sechub-notification/src/main/java/com/mercedesbenz/sechub/domain/notification/user/SignUpRequestedUserNotificationService.java

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.domain.notification.user;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Service;
+
+import com.mercedesbenz.sechub.domain.notification.email.EmailService;
+import com.mercedesbenz.sechub.domain.notification.email.MailMessageFactory;
+import com.mercedesbenz.sechub.sharedkernel.Step;
+import com.mercedesbenz.sechub.sharedkernel.messaging.UserMessage;
+import com.mercedesbenz.sechub.sharedkernel.usecases.user.UseCaseUserSignup;
+
+@Service
+public class SignUpRequestedUserNotificationService {
+
+    @Autowired
+    MailMessageFactory factory;
+
+    @Autowired
+    EmailService emailService;
+
+    @UseCaseUserSignup(@Step(number = 3, name = "Email to user", description = "A notification is send per email to user that a new user signup has been created and waits for acceptance."))
+    public void notify(UserMessage userMessage) {
+        /* build content */
+        StringBuilder emailContent = new StringBuilder();
+
+        emailContent.append("Thank you for the registration in SecHub.\n");
+        emailContent.append("Please wait of the admin's confirmation.\n");
```

Comment:
From a user perspective I would not exactly know what the next steps are based on that sentence.
In my opinion, a sentence like:

~~~
Next steps, an administrator needs to accept your registration for you to get access to SecHub.
This might take some time.
~~~

Feel free to change the message, if you think something is missing or not understandable.